src/sharedTypes.ts

```diff
@@ -1,3 +1,11 @@
+// Transcription chunks are either a take group or a paragraph of words
+export type TranscriptionChunk = TakeGroup | Word[];
+
+export interface SelectionState {
+  self: IndexRange;
+  others: Record<ClientId, IndexRange>;
```

Comment:
The tests look like they're failing because `ClientId` is not defined. What is the ffmpeg issue?